rel/mim1.vars-toml.config

```diff
@@ -9,6 +9,7 @@
 {default_server_domain, "\"localhost\""}.
 
 {mod_privacy, ""}.
```

Comment:
Do we need to have it in the config if it's a dependency of `mod_blocking`?

Reply:
Actually, not really :)

Reply:
On the other hand, isn't it clearer? If we can see in tests that mod_privacy is already expected to be enabled (big_tests aren't trying to do an rpc-start_module), we will already know this way that this module should be enabled, without having to check all dependencies.